tests/tck/features/yield/yield.feature

```diff
@@ -325,70 +325,64 @@ Feature: Yield Sentence
       """
     Then a SemanticError should be raised at runtime: `$-.abc', not exist prop `abc'
 
-  @skip
   Scenario: CalculateOverflow
     When executing query:
       """
       YIELD 9223372036854775807+1
       """
-    Then a ExecutionError should be raised at runtime: Out of range 9223372036854775807 + 1
+    Then a ExecutionError should be raised at runtime: result of (9223372036854775807+1) cannot be represented as an integer
```

Comment:
The origin error message is more clear.

Reply:
This change is to unify the error format with Neo4j